<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="n1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="n2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_ans">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>150</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="operation">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Addition</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Subraction</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Division</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Multiplication</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Operation</string>
    </property>
   </widget>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Languages">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>430</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>English</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Lang_label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>430</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Language</string>
    </property>
   </widget>
   <widget class="QLabel" name="op_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="result_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>531</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Note: Adjust the speech rate by pressing the apostrophe or semicolon key,which
 are located to the left of the Enter key. Use the Space key to replay the question,
 and employ the Shift key to hear the question in verbose mode
 use A,S,M,D keys to chnage the operation mode </string>
    </property>
   </widget>
   <widget class="QComboBox" name="Difficulty_levels">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>480</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Simple</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Easy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Medium</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hard</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Challenging</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="difficulty_label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>480</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Difficulty</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
